libs/vkd3d/resource.c

```diff
@@ -17,6 +17,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#include "vkd3d.h"
```

Comment:
Is this include accidental?

Reply:
VSCode keeps adding in random includes >.<